src/cmd/deploy.ts

```diff
@@ -142,8 +157,12 @@ export default class DefenderDeploy {
     const monitorDifference = _.differenceWith(
       monitorItems,
       Object.entries(monitors),
-      (a: DefenderMonitor, b: [string, Monitor]) =>
-        a.stackResourceId === getResourceID(getStackName(this.serverless), b[0]),
+      (a: DefenderMonitor, b: [string, Monitor | DefenderID]) => {
```

Comment:
nits: idealy they could be full names for readability (here and everywhere where variable are only 1 letters)